source/production/F0.Minesweeper.Components/Layout/Header.razor.css

```diff
@@ -0,0 +1,4 @@
+#f0-header {
+	text-align:center;
```

Comment:
Missing space after colon (Sorry 🙈🙈🙈)

Reply:
YOUR colon is missing space and that's why you can't poop! 
Kidding 😄 
I added it :)